packages/tldraw/src/lib/tools/SelectTool/childStates/PointingShape.ts

```diff
@@ -33,7 +36,8 @@ export class PointingShape extends StateNode {
 			outermostSelectingShape.id === focusedGroupId ||
 			// ...or if the shape is within the selection
 			selectedShapeIds.includes(outermostSelectingShape.id) ||
-			this.editor.isAncestorSelected(outermostSelectingShape.id) ||
+			// ...or if an ancestor of the shape is selected (except note shapes)...
+			(selectedAncestor && selectedAncestor.type !== 'note') ||
```

Comment:
let's add a TODO here about the hardcoding for `note`

Reply:
added!